contrib/timestep-demo/demo.py

```diff
@@ -19,7 +19,7 @@ def run(cmd):
 files = ["colored.dt" + str(i) + ".vtk" for i in range(20)]
 for t, f in enumerate(files):
     if not os.path.isfile(f):
-        run("eval_mesh.py bunny.vtk -o " + f + " " + str(t))
+        run("eval_mesh.py bunny.vtk -o " + f + " " + str(t) + " -d 3")
```

Comment:
Correct me if I am wrong, but isn't (and wasn't) the function argument missing in this call?